tutorials/openstack_workload_sfc.md

```diff
@@ -0,0 +1,197 @@
+
+There are two options for obtaining a security appliance image and its corresponding manager plugin. The first option is to use an appliance image and manager plugin provided by a security manager vendor compatible with OSC. The second option is to manually create them. 
+
+For this tutorial, it is assumed that the appliance image and security plugin will be manually created:  
+
+* The [`SAMPLE MANAGER PLUGIN`](https://github.com/opensecuritycontroller/security-mgr-sample-plugin) is a dummy plugin that is available along with OSC. 
+* The `SAMPLE APPLIANCE IMAGE` is a [CirrOS image](http://download.cirros-cloud.net/) that is [manually packaged](/plugins/security_mgr_plugin.md/#user-content-packaging-an-appliance). 
+ * Use the following meta.json file data when packaging the appliance image: 
+```json
+{
+	"metaDataVersion": "1.0",
+	"model": "CIRROS-TCPD",
+	"managerType": "ISM",
+	"managerVersion": "1.0",
+	"virtualizationType": "OPENSTACK",
+ 	"virtualizationVersion": "Icehouse",
+	"softwareVersion": "0.3.0.5000",
+	"imageName": "cirrosWithTcpDump.qcow2",
+	"minIscVersion": {
+		"major": 0,
+		"minor": 0,
+		"build": 1,
+		"versionStr": ""
+	},
+	"minCpus": 2,
+	"memoryInMb": 1024,
+	"diskSizeInGb": 2,
+	"encapsulationTypes": ["VLAN"],
+	"additionalNicForInspection": false
+}
+```
+
+### SDN Controller  
+
+OSC requires two components to implement traffic redirection and SDN notifications through an SDN controller, an SDN component and an SDN controller plugin.
+
+For this tutorial, it is assumed that the SDN component and SDN controller plugin will be manually created:  
+ 
+* The [`SDN CONTROLLER NSFC PLUGIN`](/plugins/plugins.md) is uploaded on OSC, enabling communication between the SDN controller and OSC.
+* The **SDN Component** is used in this tutorial is Neutron Servcice Function Chain
+
+
+## Set up OSC to Protect a Workload
+
+### 1. Upload Plugin
+
+Within OSC, navigate to **Manage** > **Plugins** using the left-hand menu. 
+* Upload the `SDN CONTROLLER NSFC PLUGIN`.  
+* Upload the `SAMPLE MANAGER PLUGIN`.  
+
+![Upload Plugins](images/add_plugins.jpg)  
+*Upload SDN Controller and Manager Plugins*
+
+### 2. Define Virtualization Connector  
+
+Using the left-hand menu, navigate to **Setup** > **Virtualization Connectors**, and then select **Add**.
+* Enter a name and then select **OPENSTACK** as the type.
+* Select **NSFC** (Network Service Function Chain Controller) as the type for the SDN Controller.
+* For the Keystone:
+	* Give the IP address of the `TENANT` environment in OpenStack.
+	* Enter `admin` as the Admin Tenant Name.
+	* Enter the OpenStack administrator credentials.
+
+![Add Virtualization Connector](images/add_VC_neutron_SFC.jpg)  
+*Add Virtualization Connector*
+
+### Define Manager Connector, Service Function, Distributed Appliance, Deployment Specification and Security Group. Please follow steps 3,4,5,6,7 from the tutorial for [protecting openstack workloads](openstack_workload.md) 

```

Comment:
might be better to add links like

### 3. Define Manager Connector -> link to (openstack_workload.md#3-define-manager-connector) etc